src/python/gudhi/off_utils.pyx

```diff
@@ -39,3 +41,17 @@ def read_points_from_off_file(off_file=''):
             raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT),
                                     off_file)
 
+@cython.embedsignature(True)
+def write_points_to_off_file(fname, points):
+    """Write points to an OFF file.
+
+    A simple wrapper for `numpy.savetxt`.
+
+    :param fname: Name of the OFF file.
```

Comment:
We should provide a link to [OFF file documentation](https://gudhi.inria.fr/python/latest/fileformats.html#off-file-format).
IIRC the syntax is:

```suggestion
    :param fname: Name of the `OFF file <fileformats.html#off-file-format>`_.
```